pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -1812,6 +1812,14 @@ in
     nvimRequireCheck = "refactoring";
   };
 
+  remote-sshfs-nvim = super.remote-sshfs-nvim.overrideAttrs {
+    dependencies = with self; [
+      telescope-nvim
+      plenary-nvim
+    ];
```

Comment:
This plugin depends on sshfs and ssh, it is probably a good idea to patch it like here:
https://github.com/NixOS/nixpkgs/blob/3745f505bd20f65c2d71c345c919ef0fb83e1201/pkgs/applications/editors/vim/plugins/overrides.nix#L1489-L1495